### PR TITLE
Cromwell: fix running watch_workflow from job

### PR DESCRIPTION
**Pull request by vladsaveliev**
Giving up fixing pickling, switching to a bash job to run watch_workflow. That's probably a bad way to do that, but I really want to unblock ourselves.

Test run: https://batch.hail.populationgenomics.org.au/batches/71003/jobs/2

### CODE REVIEW

File: analysis_runner/cromwell.py [hunk 1]
```diff
@@ -629,15 +664,3 @@ def _copy_resource_group_into_batch(
         )
 
     return output_filename
-
-
-def get_cromwell_oauth_token():
```
Comment (vivbak) [inline analysis_runner/cromwell.py:634]: Would removing this entirely cause an issue in server/cromwell.py? 
https://github.com/populationgenomics/analysis-runner/runs/6398752286?check_suite_focus=true#step:5:43

**Review by vivbak — COMMENTED** [review]
Thanks so much Vlad 🙏 🙏 🙏  Just one question from me.

File: analysis_runner/cromwell.py (hunk 1 above)
Comment (vladsaveliev) [reply analysis_runner/cromwell.py:634]: Good catch!

Comment (vladsaveliev) [reply analysis_runner/cromwell.py]: Thanks :) Fixed

**Review by vivbak — APPROVED** [review]
🙏 🙏 🙏

**Review by illusional — APPROVED** [review]
Sucks that it came to this :((

**Pull request merged**